gradle-plugin/src/test/groovy/func/GradleCompatibilitySpec.groovy

```diff
@@ -42,7 +42,7 @@ class GradleCompatibilitySpec extends BaseSpec {
         then:
         result.task(":tasks").outcome == TaskOutcome.SUCCESS
         where:
-        gradleVersion << ["5.6", "5.6.4", "6.9.1"]
+        gradleVersion << ["5.6", "5.6.4", "7.6.2"]
```

Comment:
this looks suspicious - does this mean it also tests "5.6" and "5.6.4"? Are these tests actually ran?

Reply:
I'll have a look - it does look like these unit tests may not be part of the CI build

Reply:
As we updated to 7, maybe that should change to tests major versions  5 6 7